### DISCUSSION TITLE
Dell Command update to newer version

### DISCUSSION THREAD

**Discussion by Cyber7guru** [microsoft/winget-pkgs#70819] [Q&A]
So fairly new user here but I was=checking out Dell command updates and noticed online there was a 4.6 just released at the beginning of august, but when I do winget install it gets version 4.5. Any one able to get that updated ?

**Comment by mdanish-kh** [comment]
Are you sure you see a version 4.6? I don't see the same version on the official website:
![image](https://user-images.githubusercontent.com/88161975/185461303-e0fc4904-e704-4efc-9967-f0abe5d0588c.png)

Maybe it's not available in my region yet, but if you can provide the download link, it would help getting it updated to winget. 
Also for such package upgrade cases, consider opening an [issue](https://github.com/microsoft/winget-pkgs/issues/new/choose) instead of a discussion which helps it get more attention. You can convert this discussion to an issue by clicking this button under the notifications section:
![image](https://user-images.githubusercontent.com/88161975/185462520-680938f0-ae0b-4a54-b37f-4b6bf8b4f865.png)

**Comment by Cyber7guru** [comment]
Yes under Support for Latitude 7320 | Drivers & Downloads | Dell US<https://www.dell.com/support/home/en-us/product-support/product/latitude-13-7320-2-in-1-laptop/drivers>[cid:abe20d44-0061-43b4-a4f3-5f1cf871e5b7]
Support for Latitude 7320 | Drivers & Downloads | Dell US<https://www.dell.com/support/home/en-us/product-support/product/latitude-13-7320-2-in-1-laptop/drivers>
Get drivers and downloads for your Dell Latitude 7320. Download and install the latest drivers, firmware and software.
www.dell.com

________________________________

**Reply by mdanish-kh** [reply]
Thanks for providing the download link! I've updated the manifests here in #70845 and #70846. You'll be able to download the newer versions of `Dell Command Update` and `Dell Command Update Universal` once these two PRs get merged and successfully pass the publish pipelines.

**Comment by Cyber7guru** [comment]
Does that come with an ETA?
________________________________

**Reply by mdanish-kh** [reply]
Well, on average a package takes 2-3 hours to pass the validation pipelines and checks. After which it awaits manual review from a [community moderator](https://github.com/microsoft/winget-pkgs/issues/15674). The mods are quite active and the package gets approved and updated to winget usually within the same day. 

Edit: Well what do ya know, as I was typing this message both of the PRs got manually approved from a moderator. Now it just needs to pass the validation and publish pipelines. At the moment publish pipelines are experiencing hiccups which you can track here #70728, but this package won't take long to get updated to winget.
